### PR TITLE
Updating plotting of new overdose subsets with NA values fixed 

### PR DESCRIPTION
**Pull request by Emmawang00**
Updating plotting of new overdose subsets with NA values fixed

### CODE REVIEW

File: 00_source_data/data.py [hunk 1]
```diff
@@ -111,94 +115,60 @@ def constant_states(states, num_constant_states, df_selection):
...
-final_states = list(states_set)
+################################################################################################
+# Important :
+
+# Virginia are separate states. West Virginia and Virginia being present in the dataset is not
+# a mistake.
+# References: https://www.whereig.com/usa/states/virginia/counties/
+# http://www.virginiaplaces.org/regions/westva.html
+# https://en.wikipedia.org/wiki/History_of_West_Virginia
+
```
Comment (Emmawang00) [inline 00_source_data/data.py:126]: That's a good finding! Thanks Eric

File: 00_source_data/data.py [hunk 2]
```diff
@@ -256,5 +240,5 @@ def constant_states(states, num_constant_states, df_selection):
     overdose_copy.groupby(["County Code", "Year"])["Deaths"].sum().reset_index()
 )
```
Comment (Emmawang00) [inline 00_source_data/data.py:242]: @EricR401S @lornamariak Are your overdose subsets being synchronized to each other's? As I saw in threshold.py, the used datasets are different and I'm using Lorna's ones to create plotting.

Comment (EricR401S) [reply 00_source_data/data.py:242]: @Emmawang00 I don't think so. That data.py file grabs a pre-made overdose_df csv. I don't know the code that created it or modified that csv.

Comment (EricR401S) [reply 00_source_data/data.py:242]: @Emmawang00 That's why I'm hesitant to regenerate the subsets, because that csv is ambiguous.

Comment (Emmawang00) [reply 00_source_data/data.py:242]: I think for overdose death datasets, we could refer to the latest codes created in threshold.py file, while for opioiods prescription datasets, we refer to the datasets here?

File: 00_source_data/data.py [hunk 3]
```diff
@@ -101,7 +102,10 @@ def constant_states(states, num_constant_states, df_selection):
 # Generating constant states
 states = ["Florida", "Washington", "Texas"]
 num_constant_states = 10
```
Comment (Emmawang00) [inline 00_source_data/data.py:104]: I think you changed to 12 below, isnt't it? We maybe do not need these lines of codes anymore

File: 00_source_data/data.py (hunk 2 above)
Comment (lornamariak) [reply 00_source_data/data.py:242]: the overdose csv has data type issues that I fixed in threshold.py. You can take that method and use it.

File: 00_source_data/data.py (hunk 3 above)
Comment (EricR401S) [reply 00_source_data/data.py:104]: Hi @Emmawang00 , that number depends on how many states you all have been working with. If you all have been working with files made from constant states 10, I would have to change it back to 10, so it doesn't conflict with everybody's work.

File: 00_source_data/data.py (hunk 2 above)
Comment (Emmawang00) [reply 00_source_data/data.py:242]: Sounds good!

File: 00_source_data/data.py (hunk 3 above)
Comment (Emmawang00) [reply 00_source_data/data.py:104]: Hi @EricR401S , yes that makes sense;) now we should change all the number of constant_states func to 10;)

Comment (EricR401S) [reply 00_source_data/data.py:104]: Hey @Emmawang00 , yeah, I'll have it all set up on the next push I make, I made a note of that here on Visual Studio.

Comment (EricR401S) [reply 00_source_data/data.py:104]: They will go back to 10, not 12.

**Pull request merged**